lib/pdf_fill/forms/va1010cg.rb

```diff
@@ -10,103 +10,127 @@ class Va1010cg < FormBase
       PDF_INPUT_LOCATIONS = OpenStruct.new(
```

Comment:
These changes correspond to the form fields in the pdf version of the 10-10CG. The field names were updated to match the updated versions of the fields on the new form.